app/src/main/res/layout/activity_clocks.xml

```diff
@@ -48,4 +48,15 @@
         android:padding="32dp"
         tools:text="5:00" />
 
+    <ImageButton
+        android:id="@+id/pauseButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@android:drawable/ic_media_pause"
+        android:backgroundTint="@color/colorAccent"
+        app:layout_constraintTop_toBottomOf="@+id/blackClock"
+        app:layout_constraintBottom_toBottomOf="@+id/blackClock"
+        app:layout_constraintStart_toStartOf="@+id/blackClock"
+        app:layout_constraintEnd_toEndOf="@+id/blackClock"/>
```

Comment:
This button visibility could be tested in the ClocksActivityTest 💪

Reply:
I just added the visibility test, but not in the same commit because I didn't know how to do it without messing history up.